docs/src/main/sphinx/release/release-369.md

```diff
@@ -0,0 +1,147 @@
+# Release 369 (19 Jan 2022)
+
+## General
+
+* Add support for `Pacific/Kanton` time zone. ({issue}`10679`)
+* Display `Physical input read time` using most succinct time unit in `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`10576`)
+* Fine tune request retry mechanism in HTTP event listener. ({issue}`10566`)
+* Add support for using PostgreSQL and Oracle with database backed resource
+  groups. ({issue}`9812`)
+* Remove unnecessary fine-grained spill toggles. ({issue}`10591`)
+* Remove distinction between system and user memory to simplify
+  cluster configuration. The configuration property
+  `query.max-total-memory-per-node` is removed. Use `query.max-memory-per-node`
+  instead. ({issue}`10574`)
+* Use formatting specified in the SQL standard when casting `double` and `real`
+  values to `varchar` type. ({issue}`552`)
```

Comment:
Can we elaborate - especially since not everyone has easy access to (or wants to) ANSI SQL standards.

Would "scientific notation" be correct?

Reply:
Not sure what it is .. also I think it does not belong into the release notes. If anything it should be in the docs.